treeherder/seta/settings.py

```diff
@@ -1,3 +1,9 @@
+# repos that SETA supports
+SETA_PROJECTS = [
+    'mozilla-inbound',
+    'autoland'
+]
```

Comment:
Thanks, I'll add a separate SETA_FIXED_BY_COMMIT_REPOS list, as well as a constant for the number of days to query, in settings